static/js/components/forms/validation.test.ts

```diff
@@ -115,18 +161,13 @@ describe("form validation util", () => {
             | FileConfigField
         ]
       }
-      const schema = getContentSchema(configItem)
-      expect(schema.toString()).toStrictEqual(
-        yup
-          .object()
-          .shape({
-            [partialField.name]: expectedYupField
-              // @ts-ignore
-              .label(partialField.label)
-              .required()
-          })
-          .toString()
-      )
+      const schema = getContentSchema(configItem, {})
```

Comment:
I think if we wanted to preserve the schema equality checks we probably could do so with `JSON.stringify` or `_.deepEqual` or someting, I'm a little torn on how valuable those are though

Reply:
It gets kind of tricky since there are many internal fields as well as some anonymous functions which don't compare well. I tried trimming away some of the internal fields but I feel like some of the relevant state is in a closure where we can't serialize it

Reply:
word, let's just not worry about it then 😄